e2e/resources/knative-resources.yaml

```diff
@@ -0,0 +1,118 @@
+apiVersion: eventing.knative.dev/v1
```

Comment:
shouldn't this be deployed by the m2k chart?

Reply:
In the e2e we are not using the charts from https://github.com/parodos-dev/orchestrator-helm-chart if that is what you mean
If there are any changes in the configmap or resource for instance, the chart would not yet reflects it whereas it is if we build everything from the generated files (for the workflow itself)
The only part of the chart that we may re-use would be the move2kube instance deployment (the knative function will be productized as well (https://issues.redhat.com/browse/FLPATH-898)